ghost/i18n/locales/es/search.json

```diff
@@ -1,9 +1,9 @@
 {
     "Authors": "Autores",
     "Cancel": "Cancelar",
-    "No matches found": "No se encontraron coincidencias",
+    "No matches found": "No se encontraron resultados",
     "Posts": "Publicaciones",
-    "Search posts, tags and authors": "Buscar publicaciones, etiquetas y autores",
-    "Show more results": "Mostrar más resultados",
+    "Search posts, tags and authors": "Buscar publicaciones, etiquetas y Autores",
+    "Show more results": "Muestra más resultados",
```

Comment:
Could you link a major Spanish-language website that uses Muestra for a show more results link?  I'm seeing Mostrar.